compiler/optimizer/OMRTransformUtil.hpp

```diff
@@ -188,6 +188,91 @@ class OMR_EXTENSIBLE TransformUtil
                                               TR::Block* mergeBlock,
                                               TR::CFG *cfg,
                                               bool markCold = true);
+
+#if defined(OMR_GC_SPARSE_HEAP_ALLOCATION)
+   /**
+    * \brief
+    *    Generate IL to load dataAddr pointer from the array header
+    *
+    * \param comp
+    *    The compilation object
+    *
+    * \param arrayObject
+    *    The array object node
+    *
+    * \return
+    *    IL for loading dataAddr pointer
+    */
+   static TR::Node *generateDataAddrLoadTrees(TR::Compilation *comp, TR::Node *arrayObject);
+#endif /* OMR_GC_SPARSE_HEAP_ALLOCATION */
+
+   /**
+    * \brief
+    *    Generate array element access IL for on and off heap contiguous arrays
+    *
+    * \param comp
+    *    The compilation object
+    *
+    * \param arrayNode
+    *    The array object node
+    *
+    * \param offsetNode
+    *    The offset node (in bytes)
+    *
+    * \return
+    *    IL to access array element at offset provided by offsetNode or
+    *    first array element if no offset node is provided
+    */
+   static TR::Node *generateArrayAddressTrees(
```

Comment:
Can this be named `generateArrayElementAddressTrees` (or similar) to more clearly reflect what it is doing?

Reply:
Changed helper name.